lib/howitzer/web/page.rb

```diff
@@ -162,6 +162,12 @@ def reload
         Howitzer::Log.info "Reload '#{current_url}'"
         visit current_url
       end
+
+      private
+
+      def chrome_browser?
```

Comment:
Firstly, having this code here will blow up the class. 
We already have Capybara Utils
https://github.com/strongqa/howitzer/blob/master/lib/howitzer/capybara_helpers.rb#L37
Looks like, Chrome headless is not using there and it is bug. I would extract method from there for Web page. We need to find some elegant solution for it.